crates/air-lib/trace-handler/src/merger/position_mapping.rs

```diff
@@ -14,33 +14,30 @@
  * limitations under the License.
  */
 
-use super::*;
+use super::DataKeeper;
+use super::DataPositions;
 
-pub(super) enum PrepareScheme {
+pub(super) enum PreparingScheme {
     Previous,
     Current,
     Both,
 }
 
-pub(super) fn prepare_call_result(
-    value: CallResult,
-    scheme: PrepareScheme,
-    data_keeper: &mut DataKeeper,
-) -> MergerCallResult {
+/// Prepares new_to_old_pos mapping in data keeper to keep track of value sources.
+pub(super) fn prepare_positions_mapping(scheme: PreparingScheme, data_keeper: &mut DataKeeper) {
+    // it's safe to sub 1 from positions iff scheme was set correctly
     let prev_pos = match scheme {
-        PrepareScheme::Previous | PrepareScheme::Both => Some(data_keeper.prev_slider().position() - 1),
-        PrepareScheme::Current => None,
+        PreparingScheme::Previous | PreparingScheme::Both => Some(data_keeper.prev_slider().position() - 1),
+        PreparingScheme::Current => None,
     };
 
     let current_pos = match scheme {
-        PrepareScheme::Current | PrepareScheme::Both => Some(data_keeper.current_slider().position() - 1),
-        PrepareScheme::Previous => None,
+        PreparingScheme::Current | PreparingScheme::Both => Some(data_keeper.current_slider().position() - 1),
+        PreparingScheme::Previous => None,
```

Comment:
I suggest adding `use PreparingScheme::*` at the beginning, as it will improve readability